### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by rpwils**
initial commit for the project

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+language: python
+python:
+  # We don't actually use the Travis Python, but this keeps it organized.
+  - '3.7'
+install:
+  - sudo apt-get update
+  # We do this conditionally because it saves us some downloading if the
+  # version is the same.
+  wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```
Comment (ps2) [inline .travis.yml:9]: This is just a simple parser; we shouldn't have to depend on conda (60M) for testing it.

**Review by ps2 — CHANGES_REQUESTED** [review]
Looks good as far as initial packaging goes. I think we shouldn't depend on conda for runtime or for testing, since this project has a relatively small scope.

File: .travis.yml (hunk 1 above)
Comment (rpwils) [reply .travis.yml:9]: updated

**Comment by ed-nykaza** [comment]
@ps2 can you add me to this repository

**Comment by rpwils** [comment]
> Looks good as far as initial packaging goes. I think we shouldn't depend on conda for runtime or for testing, since this project has a relatively small scope.

@ps2 would you mind reviewing once more.

**Comment by rpwils** [comment]
@ps2 @ed-nykaza can you check this PR?

**Comment by rpwils** [comment]
we will be using rpw/initial and not this branch

**Pull request closed without merging**